client/task-list/tasks/products/product-template-modal.scss

```diff
@@ -8,24 +8,33 @@ $border-color: $gray-100;
 	}
 }
 .woocommerce-product-template-modal__list {
-	.woocommerce-list {
-		margin: 0 #{-1 * $gap-large};
-		border-bottom: 1px solid $border-color;
-		border-top: 1px solid $border-color;
-	}
-	.woocommerce-list__item-inner {
-		.components-radio-control__input {
-			width: 20px;
-			height: 20px;
-			margin-right: $gap;
-			&:checked::before {
-				margin: 1px 0 0 1px;
-				width: 10px;
-				height: 10px;
-				background: var(--wp-admin-theme-color);
-			}
-			&:focus {
-				box-shadow: none;
+	.components-base-control {
+		margin: 0 -#{$gap-large};
+		.components-base-control__field {
+			.components-radio-control__option {
+				display: flex;
+				text-decoration: none;
+				width: 100%;
+				align-items: center;
+				padding: $gap $gap-large;
+				margin-bottom: 0;
+				box-sizing: border-box;
+				border-bottom: 1px solid $gray-100;
+				&:first-child {
+					border-top: 1px solid $gray-100;
+				}
+				.components-radio-control__input {
+					margin-right: $gap;
+					width: 20px;
+					height: 20px;
+					margin-right: $gap;
+					&:checked::before {
+						box-sizing: border-box;
+					}
+					&:focus {
+						box-shadow: none;
+					}
+				}
```

Comment:
Why does all of this need to be specified? I would think we would get almost all of what we need just from the core component.